docs/reference/commandline/service_create.md

```diff
@@ -797,6 +798,7 @@ The options you can specify are:
     port for both protocols, specify the `-p` or `--publish` flag twice.
   </p></td>
 </tr>
+
```

Comment:
Can you remove this line change and amend your commit?

Reply:
Some swearing at text editors adding and removing spaces from ends of lines later - all done

Reply:
Oooh, sorry for that, thanks so much for fixing!